test/androidTest/java/org/thoughtcrime/securesms/VideoCompressionOptionMenuTest.java

```diff
@@ -0,0 +1,115 @@
+package org.thoughtcrime.securesms;
+
+
+import android.support.test.espresso.ViewInteraction;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.test.suitebuilder.annotation.LargeTest;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.InstrumentationRegistry.getInstrumentation;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.contrib.RecyclerViewActions.actionOnItemAtPosition;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withClassName;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.is;
+
+@LargeTest
+@RunWith(AndroidJUnit4.class)
+public class VideoCompressionOptionMenuTest {
+
+    @Rule
+    public ActivityTestRule<ConversationListActivity> mActivityTestRule = new ActivityTestRule<>(ConversationListActivity.class);
+
+    @Test
+    public void videoCompressionOptionMenuTest() {
+        //open action bar menu
+        openActionBarOverflowOrOptionsMenu(getInstrumentation().getTargetContext());
+        //open settings tab
+        openSettingsMenu();
+        //open chat and media settings tab
+        openChatAndMediaMenu();
```

Comment:
Should think of using the util classes/moving these to the util classes.

Reply:
good idea

Reply:
@benrs Is there an util class that already has general helper methods for expresso?